client/my-sites/theme/main.jsx

```diff
@@ -240,7 +241,25 @@ const ThemeSheet = React.createClass( {
 		return <ThemeDownloadCard theme={ this.props.id } href={ this.props.download } />;
 	},
 
-	render() {
+	renderEmptyContent() {
+		const emptyContentTitle = i18n.translate( 'Looking for great WordPress designs?', {
+			context: 'Message displayed when requested theme was not found',
+		} );
+		const emptyContentMessage = i18n.translate( 'Check our theme showcase', {
+			context: 'Message displayed when requested theme was not found',
+		} );
+
+		return(
+			<Main className="themes__sheet_404">
+				<EmptyContentComponent title={ emptyContentTitle } line={ emptyContentMessage }/>
+				<Button primary href="/design">
+					{ i18n.translate( 'View the showcase' ) }
+				</Button>
```

Comment:
Just use `EmptyContent`'s `action` and `actionURL` props instead of a `Button`, see https://github.com/Automattic/wp-calypso/tree/master/client/components/empty-content#properties :-)